docs/hdf5_format.rst

```diff
@@ -233,9 +227,9 @@ Rational
 
 It is our experience that not having the topology stored in the same
 file as the the trajectory's coordinate data is a pain -- it's just really
-inconvenient. And generally, the trajectories are long enough that it
-doesn't take up much incremental storage space to store the topology in
-there too. The topology is not that complicated.
+inconvenient. Furthermore, the topology takes up a miniscule quantity of
```

Comment:
This seems better, but it is almost more of a meaning change while other edits were corrections.